Src/DBProj/Tables/Products.sql

```diff
@@ -8,6 +8,7 @@
     [ModifiedOn]  DATETIME     NULL,
     [ModifiedBy]  INT          NULL,
     [IsActive]    BIT          NOT NULL,
+    [ProductCategory] VARCHAR(100) NOT NULL, 
```

Comment:
@MuhammadZain795 
Please Create a separate table dbo.ProductCategory and use its ProductCategoryID as reference key in Product table.

Reply:
ok Sir! @bilalshahzad139